Formula/passenger.rb

```diff
@@ -18,9 +19,13 @@ class Passenger < Formula
   depends_on "openssl@1.1"
   depends_on "pcre"
 
+  patch do
+    url "https://github.com/phusion/passenger/commit/e512231f.patch?full_index=1"
+    sha256 "9f39f5c1c8b68516f7bac0ba07921144a5de30b6a72ef2423ea83a77d512bea8"
+  end
```

Comment:
What's this and the `passenger_temp_path` about? Could use a comment.

Reply:
True, it's to address a failure seen here: https://github.com/Homebrew/homebrew-core/pull/45685#issuecomment-550307627 (sandbox violation) I started this PR to test the ruby headers fix, but when that wasn't working I just started using it to test the temp path issue instead then circled back to the headers thing now that the temp path part is working.

Reply:
Added comment